### PR TITLE
Update LIP 0063 "Define mainnet configuration and migration for Lisk Core v4" with respect to the last module updates

### PR DESCRIPTION
**Pull request by sergeyshemyakov**

### CODE REVIEW

File: proposals/lip-0063.md [hunk 1]
```diff
@@ -365,19 +375,20 @@ def addTokenModuleEntry():
     tokenObj.supplySubstore = createSupplySubstoreArray()
     tokenObj.escrowSubstore = []
     tokenObj.availableLocalIDSubstore = {}
+    tokenObj.supportedTokensSubstore = [{"chainID": MAINCHAIN_ID, "supportedTokenIDs": [TOKEN_ID_LSK]}]
```
Comment (janhack) [inline proposals/lip-0063.md:378]: Checking the function `isTokenSupported` in the Token module and rationale section, I think this is not needed (although it is also not incorrect).  Not sure about this one.

**Review by janhack — APPROVED** [review]
I have one small optional discussion point, but the PR looks good so I already approve it.

File: proposals/lip-0063.md (hunk 1 above)
Comment (sergeyshemyakov) [reply proposals/lip-0063.md]: Agree, removed. I think, we should not put LSK into supported tokens substore.

**Review by vjaiman — APPROVED** [review]
Looks all good.

**Review by karmacoma — APPROVED** [review]

**Pull request merged**